packages/device_info/device_info/pubspec.yaml

```diff
@@ -19,14 +22,15 @@ flutter:
 dependencies:
   flutter:
     sdk: flutter
-  device_info_platform_interface: ^1.0.0
+  device_info_platform_interface:
+    path: ../device_info_platform_interface
```

Comment:
We are going with git dependencies, but feel free to change it after this PR is merged.